scripts/completions/install.py

```diff
@@ -189,6 +202,101 @@ def _update_rc(
         assert mode == "uninstall"
 
 
+def _update_conda_scripts(
+    completions_dir: pathlib.Path,
+    mode: ConfigureMode,
+) -> None:
+    """Try to add a `source /.../completions/setup.{shell}` line automatically to conda's activation scripts.
+
+    Args:
+        completions_dir: Path to location of this script.
+        mode: Install or uninstall completions.
+    """
+
+    # Install or uninstall `source_line`.
+    source_lines = _get_source_script(completions_dir, None)
+
+    conda_path = pathlib.Path(os.environ["CONDA_PREFIX"])
+    activate_path = conda_path / "etc/conda/activate.d/nerfstudio-activate.sh"
+    if mode == "uninstall":
+        if activate_path.exists():
+            os.remove(activate_path)
+        CONSOLE.log(f":broom: Existing completions uninstalled from {conda_path}.")
+    elif mode == "install":
+        # Install completions.
+        with activate_path.open("w+", encoding="utf8") as f:
+            f.write(source_lines)
+        # TODO: handle also the deactivate case
+        CONSOLE.log(
+            f":person_gesturing_ok: Completions installed to {conda_path}. {_exclamation()} Reactivate the environment"
+            " to try them out."
+        )
+    else:
+        assert_never(mode)
+
+
+def _get_conda_path() -> Optional[pathlib.Path]:
+    """
+    Returns the path to the conda environment if
+    the nerfstudio package is installed in one.
+    """
+    conda_path = None
+    if "CONDA_PREFIX" in os.environ:
+        # Conda is active, we will check if
+        # the Nerfstudio is installed in the conda env
```

Comment:
nit: line break / comment formatting is a little weird

maybe we could also make the comment style consistent within this file, like ending with periods etc?

Reply:
Better?

Reply:
Yes, thanks :)